### PR TITLE
i422 - back up teamcity configuration

### PR DESCRIPTION
**Pull request by richfitz**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-422

This is manually checked out on support so that I could test it.  Once this is merged in I'll get this added via montagu-backup

### CODE REVIEW

File: scripts/write-cron-teamcity-backup-sync.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+if [ ! -f scripts/sync-backups.sh ]; then
+    echo "Run this from the montagu-ci directory"
+    exit 1
+fi
+
+PATH_VAGRANT=$PWD
+DEST=/etc/cron.daily/teamcity-backup-sync
+
+cat <<EOF > $DEST
```
Comment (MartinEden) [inline scripts/write-cron-teamcity-backup-sync.sh:31]: What does this black magic do?

**Review by MartinEden — CHANGES_REQUESTED** [review]
In general, I feel like I kind of know what `write-cron-teamcity-backup-sync.sh` is doing, but I don't really get how it's doing it. Could you add some comments to that file?

I think what you are doing is `cat`ing an inline crontab into the VM. But I don't understand what `PATH_VAGRANT` is doing.

File: scripts/write-cron-teamcity-backup-sync.sh (hunk 1 above)
Comment (richfitz) [reply scripts/write-cron-teamcity-backup-sync.sh:31]: I've added an explanation but I'm not sure what part is the black magic!

Comment (richfitz) [reply scripts/write-cron-teamcity-backup-sync.sh:31]: Ah, given

> I think what you are doing is cating an inline crontab into the VM. But I don't understand what PATH_VAGRANT is doing.

I've added a further explanation about what is being affected here - this is run on the _host_ machine

**Review by MartinEden — APPROVED** [review]

**Pull request merged**